packages/coverage-v8/src/browser.ts

```diff
@@ -71,9 +71,5 @@ function filterResult(coverage: ScriptCoverage['result'][number]): boolean {
     return false
   }
 
-  if (coverage.url.includes('?browserv=') || coverage.url.includes('&browserv=')) {
-    return false
-  }
-
```

Comment:
Need to check with Storybook folks to see that this doesn't break coverage reports there. This was used to exclude `*.stories.*` from appearing in coverage report.

Reply:
Storybook can now exclude story files inside the `configureVitest` hook

Reply:
Yep, that works! Using this in fresh Storybook project fixes the issue:

```ts
      {
        name: "exclude-stories",
        configureVitest(context) {
          if (context.vitest.config.coverage.enabled) {
            context.vitest.config.coverage.exclude.push("**/*.stories.**");
          }
        },
```